pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Allows to connect to the cloud powered by Pasqal to run sequences."""
+from typing import Dict, List, Mapping, Optional, Union
+
+import sdk
+from numpy.typing import ArrayLike
+
+from pulser import Sequence
+from pulser.devices import Device
+from pulser.register import QubitId
+
+
+class PasqalCloud:
+    """Manager of the connection to the cloud powered by Pasqal.
+
+    The cloud connection enables to run sequences on simulators or on real
+    QPUs.
+
+    Args:
+        client_id: client_id of the API key you are holding for Pasqal
+            cloud.
+        client_secret: client_secret of the API key you are holding for
+            Pasqal cloud.
+        endpoints: Optionally, Pasqal cloud connection URLs.
+        webhook: Optionally, webhook.
```

Comment:
This is not relevant for most people and doesn't have to be available here perhaps?

The webhook is an url that our webservice can send job results to.

Reply:
Is it the case for `endpoints` too? For these type of "internal" features, perhaps we could hide them under `kwargs` so that they are still available but not visible to all users

Reply:
True, the endpoints would be specified to access other environments than production.

the `kwargs` approach is probably the best idea indeed, as at least endpoints is not unlikely to be used internally.

Reply:
I like the `kwargs` approach as well.